marathon_lb.py

```diff
@@ -72,6 +72,9 @@ def __init__(self, appId, servicePort, healthCheck):
         self.servicePort = servicePort
         self.backends = set()
         self.hostname = None
+        self.proxypath = ''
```

Comment:
There is no particular reason. If you want / expect a particular style, then I would suggest that documenting it would be a good move.